tgui/packages/tgui/interfaces/GasAnalyzer.tsx

```diff
@@ -29,11 +54,85 @@ export const GasAnalyzerContent = (props, context) => {
   );
 };
 
+export const GasAnalyzerHistory = (props, context) => {
+  const { act, data } = useBackend<GasAnalyzerHistoryData>(context);
+  const { historyGasmixes, historyViewMode, historyIndex } = data;
+  return (
+    <Section
+      scrollable
+      fill
+      title="Scan History"
+      buttons={
+        <Button
+          icon={'trash'}
+          tooltip="Clear History"
+          onClick={() => act('clearhistory')}
+          textAlign="center"
+          disabled={historyGasmixes.length === 0}
+        />
+      }>
+      <LabeledList.Item label="Mode">
+        <Flex inline width="50%">
```

Comment:
use stack, and please do not set width like this, use grow and fill props. [component reference](https://github.com/tgstation/tgstation/blob/master/tgui/docs/component-reference.md)